app/views/step_by_step_pages/reorder.html.erb

```diff
@@ -0,0 +1,67 @@
+<%
+  links = [
+    {
+      text: 'Step by step publisher',
+      href: step_by_step_pages_path
+    },
+    {
+      text: @step_by_step_page.title,
+      href: step_by_step_page_path(@step_by_step_page)
+    },
+    {
+      text: 'Reorder steps'
+    }
+  ]
+%>
+
+<%= render 'shared/steps/step_breadcrumb', links: links %>
+
+<%= render 'shared/steps/form_notice', message: notice, status: "success" %>
+
+<h1>
+  <%= @step_by_step_page.title %><br />
+  <small>Reorder steps</small>
+</h1>
+
+<div class="row">
+  <div class="col-md-7">
+    <p>Use the up/down controls on the right to reorder steps. Step numbers will be automatically updated on save.</p>
+
+    <%
+      step_number = 1
+      steps_length = @step_by_step_page.steps.length - 1
+    %>
+    <% @step_by_step_page.steps.each_with_index do |step, step_index| %>
```

Comment:
Could we add a message here if you have no steps?  It looks a bit weird otherwise.